docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Meta
```

Comment:
I wonder if this title is necessary? It looks a little strange in the rendered output

![image](https://user-images.githubusercontent.com/6527489/64008781-ad5c5180-cb0e-11e9-87f6-7e34e0065810.png)

Reply:
The default (which appears if you don't have a top level heading) is "Home". I'll change to that.

Reply:
9e8a42a